media/js/firefox/tracking-protection-tour.js

```diff
@@ -138,6 +142,19 @@ if (typeof Mozilla === 'undefined') {
         $('#info-panel header > button').on('click.tp-tour', TPTour.step4);
         $(document).on('visibilitychange.tp-tour', TPTour.handleVisibilityChange);
         $('#reload-btn').on('click.tp-tour', TPTour.restartTour);
+        $(window).on('resize.tp-tour', TPTour.handleResize);
+    };
+
+    /**
+     * Tempoaray workaround for resizing the window (Bug 1188400)
+     */
+    TPTour.handleResize = function() {
+        if (TPTour.state === 'step1' && TPTour.panelVisible) {
+            clearTimeout(_resizeTimeout);
+            Mozilla.UITour.hideInfo();
+            TPTour.panelVisible = false;
+            _resizeTimeout = setTimeout(TPTour.showTourStep, 300);
+        }
```

Comment:
We need the handler to be pretty snappy, but now that you mention it I think we can optimize this a bit more :+1: